setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
This file is used to set different settings than the *flake8*'s defaults.
We will prefer to stick with `max-line-length` as 80, but will every time run:
```
pipenv run flake8 --max-line-length 120
```

Please remove this file, so that `max-line-length` will be 80 as the default value.